test/images/clusterapi-tester/bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash -x
+
+BIN=clusterapi-tester
+SRCS=clusterapi-tester.go
+ARCH=$1
+TARGET=$2
+CGO_ENABLED=0 GOARM=${GOARM} GOARCH=${ARCH} go build -a -installsuffix cgo --ldflags '-w' -o ${TARGET}/${BIN} ./${SRCS}
```

Comment:
I need source file only for few images where I need to build multiple binaries in the same folder.